components/molecules/FilterHeader/filter-header.tsx

```diff
@@ -40,20 +37,19 @@ const HeaderFilter = () => {
   return (
     <>
       <div className="header-image mr-2 p-2 min-w-[130px] ">
-        <ContextThumbnail size={120} ContextThumbnailURL={isHacktoberfest ? Thumbnail.src : ""}></ContextThumbnail>
+        <ContextThumbnail size={120} ContextThumbnailURL={getTopicThumbnail(filterName as interestsType)}></ContextThumbnail>
       </div>
       <div className="header-info md:truncate flex flex-col grow justify-center p-2">
         <Title level={1} className="!text-3xl font-semibold tracking-tight text-slate-900">
-          {isHacktoberfest ? "Hacktoberfest 2022" : topicNameFormatting(filterName as string)}
+          {topicNameFormatting(filterName as string)}
         </Title>
         <Text className="mt-1 !text-base   text-slate-500">
-          Insights on GitHub repositories{" "}
-          {isHacktoberfest ? "opted into the largest open source hackathon." : `using the ${filterName} topic.`}
+          {`Insights on GitHub repositories using the ${topicNameFormatting(filterName as string)} topic.`}
         </Text>
         <div className="flex mt-4 items-center gap-2">
           <FilterCardSelect
             selected={filterName as string}
-            options={topicOptions}
+            options={topicOptions as unknown as []}
```

Comment:
`topicOptions as unknown as []`
You're forced to do this as `topicOptions` now returns a readonly string[], while `options` asks only for a normal string[]. `options` doesn't change in the string array though, so instead of doing this type-casting, go to the definition of the prop `options` and convert its type to `readonly string[]`.

Reply:
@Deadreyo  The second approach seems sustainable. After doing some quick research I got to know that

when we use as unknown as string[], we are telling TypeScript to trust that the readonly array is actually a string[] array at runtime, even though TypeScript cannot verify this. This can lead to runtime errors if the actual type of the array is not string[].

So instead of using  `as unknown as string[]` I used the array spreading technique here:

`const interests = ["javascript", "python", "java", "typescript", "csharp", "cpp", "php", "c", "ruby" , "ai", "ml", "react"] as const;`

`export const getInterestOptions = () => {
  return [...interests];
};`

Reply:
That's pretty good, nice move!